routing/pathfind_test.go

```diff
@@ -232,6 +233,13 @@ func parseTestGraph(path string) (*testGraphInstance, error) {
 			return nil, err
 		}
 
+		if bytes.Compare(node1Bytes, node2Bytes) == 1 {
```

Comment:
style nit: compare `> 0`

Reply:
Leave it as is, because the go doc on `bytes.Compare` explictly mentions the special values.